packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -490,39 +489,6 @@ const schema: SchemaType<"Users"> = {
     },
     // unique: true // note: find a way to fix duplicate accounts before enabling this
   },
-  // The user's profile URL slug // TODO: change this when displayName changes
-  // Unique user slug for URLs, copied over from Vulcan-Accounts
-  slug: {
-    type: String,
-    optional: true,
-    canRead: ['guests'],
-    canUpdate: ['admins'],
-    order: 40,
-    group: formGroups.adminOptions,
-    
-    onCreate: async ({ document: user }) => {
-      // create a basic slug from display name and then modify it if this slugs already exists;
-      const displayName = createDisplayName(user);
-      const basicSlug = slugify(displayName);
-      return await getUnusedSlugByCollectionName('Users', basicSlug);
-    },
-    onUpdate: async ({data, oldDocument}) => {
-      if (data.slug && data.slug !== oldDocument.slug) {
-        const slugLower = data.slug.toLowerCase();
```

Comment:
It doesn't seem like we preserved the toLower in the new collection hooks abstraction (for users)